node/runtime/pangolin/Cargo.toml

```diff
@@ -144,7 +144,7 @@ std = [
 	"scale-info/std",
 	# darwinia-network
 	"bsc-primitives/std",
-	"common-runtime/std",
+	"drml-common-runtime/std",
```

Comment:
Little format issue.

Reply:
What issue?

Reply:
By convention, this dependency should be moved down to `drml-common-primitives/std"` position.